### ISSUE TITLE
Bug: Icon image has a default alt text of "Icon"

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#3010]
The `<Icon />` component has a default value of "Icon" for alt text. It should be empty alt text as it provides no value in terms of accessibility and typically you don't have alt text that says this is an image or icon.

```diff
const Icon: React.FC<IconProps> = ({ onClick, className, IconImage, alt, size = 16 }) => {
  return (
    <div className={`${className ? className : ""} inline-flex`}>
      <Image
        onClick={onClick}
        className="items-center justify-center "
-        alt={alt ? alt : "Icon"}
+        alt={alt ?? ""}
        width={size}
        height={size}
        src={IconImage}
        style={{
          maxWidth: "100%",
          height: "auto",
        }}
      />
    </div>
  );
};
```

This is the file and line to change https://github.com/open-sauced/app/blob/4983cd8ebdb4f177f82b8135121a4db29a15d4c6/components/atoms/Icon/icon.tsx#L18

I came across this while writing end to end tests which actually caused my test to fail when searching for button text via, e.g. `await expect(page.getByRole("button", { name: "Connect with GitHub", exact: true })).toBeVisible();`

**Comment by nickytonline** [comment]
@Jenna59, would you like to work on this as a first contribution to OpenSauced?

**Comment by nickytonline** [comment]
You need to assign yourself to the issue @Re1gner

**Comment by Re1nGer** [comment]
.take

**Comment by Re1nGer** [comment]
@nickytonline thanks for guidance, now the issue is assigned to me
